Cargo.lock

```diff
@@ -171,6 +171,7 @@ dependencies = [
  "memmap 0.6.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "notify 4.0.3 (registry+https://github.com/rust-lang/crates.io-index)",
  "ole32-sys 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "open 1.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Can you remove `Cargo.lock` from your PR? I'll update it later myself when I'll be releasing a new version of `cargo-web`.

Reply:
Sure